pkg/mirrorregistries/generator_test.go

```diff
@@ -60,4 +72,51 @@ var _ = Describe("MirrorRegistriesConfig", func() {
 		Expect(err).NotTo(HaveOccurred())
 		Expect(string(contents)).Should(Equal("some ca data"))
 	})
+
+	It("check if mirrors configured - CA and registry.conf don't exist", func() {
+		m := mirrorRegistriesConfigBuilder{
+			MirrorRegistriesConfigPath:      "/tmp/this-file-for-sure-does-not-exist",
+			MirrorRegistriesCertificatePath: "/tmp/this-file-for-sure-does-not-exist",
+		}
+		res := m.IsMirrorRegistriesConfigured()
+		Expect(res).Should(Equal(false))
+	})
+
+	It("check if mirrors configured - registry.conf exists without registries, CA exists", func() {
+		tempRegistriesFile, err := ioutil.TempFile(os.TempDir(), "registries.*.conf")
+		defer os.Remove(tempRegistriesFile.Name())
+		Expect(err).NotTo(HaveOccurred())
+		_, _ = tempRegistriesFile.WriteString(configWithoutMirrors)
+
+		tempCAFile, err := ioutil.TempFile(os.TempDir(), "ca.*.crt")
+		defer os.Remove(tempCAFile.Name())
+		Expect(err).NotTo(HaveOccurred())
+		_, _ = tempCAFile.WriteString("some random CA certificate")
+
+		m := mirrorRegistriesConfigBuilder{
+			MirrorRegistriesConfigPath:      tempRegistriesFile.Name(),
+			MirrorRegistriesCertificatePath: tempCAFile.Name(),
+		}
+		res := m.IsMirrorRegistriesConfigured()
+		Expect(res).Should(Equal(false))
+	})
+
+	It("check if mirrors configured - registry.conf has registries, CA exists", func() {
+		tempRegistriesFile, err := ioutil.TempFile(os.TempDir(), "registries.*.conf")
```

Comment:
This function is deprecated. ref: https://pkg.go.dev/io/ioutil@go1.19#TempFile

Can you use `os.CreateTemp` instead?